pom.xml

```diff
@@ -354,6 +355,25 @@
                                         </test>
                                     </tests>
                                 </member>
+                                <member>
+                                    <name>CXF</name>
+                                    <bom>io.quarkiverse.cxf:quarkus-cxf-bom:${quarkus-cxf.version}</bom>
+                                    <release>
+                                        <lastDetectedBomUpdate>io.quarkus.platform:quarkus-cxf-bom:3.0.0.CR1</lastDetectedBomUpdate>
```

Comment:
@aloubyansky what should lastDetectedBomUpdate actually be for a newly added member?

Reply:
It simply should not be configured.

Reply:
How simple!